MvvmCross/Binding/Bindings/Source/Leaf/MvxLeafPropertyInfoSourceBinding.cs

```diff
@@ -61,7 +61,7 @@ public override void SetValue(object value)
 
             if (!PropertyInfo.CanWrite)
             {
-                MvxBindingLog.Warning("SetValue ignored in binding - target property is readonly");
+                MvxBindingLog.Warning("SetValue ignored in binding - target property {0}.{1} is readonly", PropertyInfo.DeclaringType?.Name, PropertyName);
```

Comment:
Is there a reason why we log an Error when GetValue is called when not CanRead but we raise a Warning when SetValue called and not CanWrite?

Reply:
No idea. I don't have deeper insight into this part of the code.

Reply:
Ok, will approve as is. Can adjust warning/error later if an issue for anyone